docs/plugins.jetbrains.com/general_usage_instructions.html

```diff
@@ -6,15 +6,15 @@
     Open Git Machete tab:
     <ul>
       <li>Navigate <b>Git</b> (tool window) -> <b>Git Machete</b> (tab)</li>
-      <li>Or use key shortcut: Ctrl + Alt + Shift + M (Command + Option + Shift + M on Mac OS)</li>
```

Comment:
Pls update the plugin site while you're here

Reply:
Will do once approved